### ISSUE TITLE
Add a list of installed software and versions

### ISSUE THREAD

**Issue by apertome** [perfsonar/toolkit#54]
Add a list of installed software and versions, possibly in the sidebar

This suggests we: 
* add a "get_software" webservice method that returns a more detailed list of relevant software and versions
* add the get_software method output to the get_summary method
* add the get_software call to the JS HostStore
* create a new widget and add to the sidebar (or wherever). we'll probably want it set so you have to click to expand this info, as it may be a long list.

**Comment by apertome** [comment]
low priority, may or may not happen for 3.5

**Comment by apertome** [comment]
Changing to 3.6. We have the method but not sure where to put this info just yet.

**Comment by apertome** [comment]
I thought we had a get_software method already, but we don't. I don't know why I thought we had this. May be worth adding at some point.

**Comment by mfeit-internet2** [comment]
Should probably limit access to anything that can enumerate what's installed to trusted users to make it more difficult to probe for known vulnerabilities.

**Comment by apertome** [comment]
Good point, Mark, that was the intent, but it's good to be explicit about that.
